modules/data-streams/src/internalClusterTest/java/org/elasticsearch/datastreams/LazyRolloverDuringDisruptionIT.java

```diff
@@ -84,7 +99,7 @@ public void onFailure(Exception e) {
         }
 
         // End the disruption so that all pending tasks will complete
-        masterNodeDisruption.stopDisrupting();
+        safeAwait(barrier);
 
         // Wait for all the indexing requests to be processed successfully
         countDownLatch.await();
```

Comment:
unrelated nit: slight preference for `safeAwait(countDownLatch)` here, no sense in waiting forever.